storage/namespace.go

```diff
@@ -728,6 +732,51 @@ func (n *dbNamespace) Flush(
 	return res
 }
 
+func (n *dbNamespace) Snapshot(blockStart, callStart time.Time, flush persist.Flush) error {
+	n.RLock()
+	if n.bs != bootstrapped {
+		n.RUnlock()
+		n.metrics.snapshot.ReportError(n.nowFn().Sub(callStart))
```

Comment:
think you need a separate call start which calls `n.nowFn()` at the start of this function

Reply:
yep just noticed that, thanks